setup.py

```diff
@@ -28,6 +28,7 @@
     ],
     setup_requires=[
         "nose>=1.0",
+        "mock>=1.0.1",
```

Comment:
You are right. I didn't know this, but [the docs](http://nose.readthedocs.io/en/latest/api/commands.html) make it clear that once you specify `"nose>=1.0"` in `setup_requires`, the `python setup.py nosetests` command will install everything in `tests_require`. Good catch. I'll remove it.